src/tscore/ConsistentHash.cc

```diff
@@ -47,7 +47,7 @@ ATSConsistentHash::insert(ATSConsistentHashNode *node, float weight, ATSHash64 *
   if (h) {
     thash = h;
   } else if (hash) {
-    thash = hash;
+    thash = &*hash;
```

Comment:
There are a couple of these uses of `&*` vs `.get()`.  Why not use `.get()` here as well? Its more clear imo than `&*`.